stdlib/Random/docs/src/index.md

```diff
@@ -66,6 +71,18 @@ Random.MersenneTwister
 Random.RandomDevice
 ```
 
+## Cryptographically secure random numbers
+
+The Julia standard library does not provide a native cryptographically secure random number generator. For most applications,
+we recommend to use the operating system `RandomDevice()`. This should be initialized via a module-level
+`const SECURE_RANDOM = Random.RandomDevice()` and subsequently used via e.g. `gen_nonce() = rand(SECURE_RANDOM, UInt128)`.
+
+It is important to initialize the `RandomDevice()` on the module level, and not use e.g.
+`bad_gen_nonce() = rand(Random.RandomDevice(), UInt128)`, in order to not cause performance degradation and an eventual
+`ERROR: SystemError: opening file "/dev/urandom": Too many open files`.
+
+The `RandomDevice` reads and caches `/dev/urandom` on Unix-derived systems, and uses appropriate `Ntsecapi.h` calls on Windows.
```

Comment:
I will trust you on this fact concerning Windows, but I infer from the name of the file that it's a CSPRNG?